app/code/Magento/Swatches/Block/Product/Renderer/Configurable.php

```diff
@@ -183,6 +183,7 @@ public function getJsonSwatchConfig()
                 );
             }
         }
+        $config['additional_data'] = $attributesData;
```

Comment:
It's better to include only the necessary information (update_product_preview_image) to minimize the output size

Reply:
Hello @sivaschenko 
I have update app/code/Magento/Swatches/Block/Product/Renderer/Configurable.php
and pass the only additional_data to minimize output size